### PR TITLE
PAS-132: Return create/update responses for all documents

### PR DESCRIPTION
**Pull request by isaiahdoyle**
In a previous ticket I wasn't aware of the importance of returning the result of creating and updating documents, so this fixes that issue by returning the created/updated document in each API response.

### CODE REVIEW

File: src/routers/course.ts [hunk 1]
```diff
@@ -36,9 +36,8 @@ courseRouter.post('/', async (req, res) => {
 				owner: userID,
 				name: req.body.name,
 			});
-			await create(course);
-			console.log('Post Course');
-			res.send('Post courseRouter: ' + req.body.name);
+			const created = await create(course);
+			res.json(created);
```
Comment (isaiahdoyle) [inline src/routers/course.ts:40]: Are there any objections to using `res.json` here instead of calling `created.toJSON()`?

Comment (BraidenCutforth) [reply src/routers/course.ts:40]: This is fine. I looked up the documentation for res.json and it called `JSON.stringify` internally, and `JSON.stringify` will try to call the `toJSON` method on the object if it exists, which it does.

**Review by BraidenCutforth — APPROVED** [review]
Just a couple changes mentioned

**Review by seiyaterada — APPROVED** [review]
Looks good!

**Pull request merged**